api/types.go

```diff
@@ -215,7 +215,6 @@ type Options struct {
 	TopK             int      `json:"top_k,omitempty"`
 	TopP             float32  `json:"top_p,omitempty"`
 	MinP             float32  `json:"min_p,omitempty"`
-	TFSZ             float32  `json:"tfs_z,omitempty"`
```

Comment:
Presumably we should at least doc somewhere that this has been removed from the API. Right now, it might be a behavior change if someone is specifying it and we start silently ignoring it.